pkgs/games/anki/sync-server.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, rustPlatform
+, anki
+
+, openssl
+, pkg-config
+, protobuf
+}:
+
+rustPlatform.buildRustPackage {
+  pname = "anki-sync-server";
+  version = anki.version;
+  src = anki.src;
+  cargoLock = anki.cargoLock;
+
+  # only build sync server
+  cargoBuildFlags = [
+    "--bin"
+    "anki-sync-server"
+  ];
+
+  nativeBuildInputs = [ protobuf pkg-config ];
+  env = {
+    PROTOC = "${protobuf}/bin/protoc";
+  };
```

Comment:
nonblocking nit:

```suggestion
  env.PROTOC = lib.getExe protobuf;
```

Reply:
Thanks, I knew libExe existed but couldn't remember the syntax!